plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/BuildPath.java

```diff
@@ -218,23 +214,6 @@ private static IClasspathEntry makeClasspathEntry(Library library) throws CoreEx
         false);
   }
 
-  public static void runContainerResolverJob(IJavaProject javaProject) {
-    IEclipseContext context = EclipseContextFactory.getServiceContext(
-        FrameworkUtil.getBundle(BuildPath.class).getBundleContext());
-    final IEclipseContext childContext =
-        context.createChild(LibraryClasspathContainerResolverJob.class.getName());
-    childContext.set(IJavaProject.class, javaProject);
-    Job job =
-        ContextInjectionFactory.make(LibraryClasspathContainerResolverJob.class, childContext);
```

Comment:
Not sure, but maybe there is some setting related to this injection stuff in `plugin.xml` that we can remove?